modules/nw-ipfailover-configuration.adoc

```diff
@@ -41,7 +39,62 @@ $ oc adm policy add-scc-to-user privileged -z ipfailover
 ----
 $ oc adm policy add-scc-to-user hostnetwork -z ipfailover
 ----
+
+. On {rh-openstack-first}, complete the following steps to make the failover VIP addresses reachable on {rh-openstack} ports: 
```

Comment:
@dfitzmau
i will suggest you to add the below instead 

`3. In the openstack cluster we need to find the 'allowed_address_pairs' using the openstack cli and then add one of IP address among the 'allowed_address_pairs' to the env variable 'OPENSHIFT_HA_VIRTUAL_IPS' in the IPFailover deployment YAML

Prerequest:- Need to install python-openstackclient and download the openstack rc file.
eg-: a. How to get the IP address for the VIP

```
           $ openstack port show <cluster_name>  -c allowed_address_pairs
           +-----------------------+-----------------------------------------------------------+
| Field                 | Value                                                     |
+-----------------------+-----------------------------------------------------------+
| allowed_address_pairs | ip_address='192.168.0.5', mac_address='fa:16:3e:5d:f0:2d' |
|                       | ip_address='192.168.0.7', mac_address='fa:16:3e:5d:f0:2d
```
    
    b. Either add VIP to the allowed pair like below or use one this above IP address as VIP
`$ openstack port set <cluster_name> --allowed-address ip-address=192.168.0.5,mac-address=fa:16:3e:5d:f0:2d'`